WebRoot/skins/_base/base4/skin.properties

```diff
@@ -626,13 +630,14 @@ TaskbarItem-selected            = @ToolbarColor@
 Select                      = @SelectHeight@ @ActiveCursor@
 SelectHeight                = @WidgetHeight@
 SelectSpacing               =
-SelectPadding               = 5px
+SelectPadding               = @WidgetInternalPadding@
+SelectIcon                  = padding: 0 @SelectPadding@;
 SelectTitle                 = padding:0 @SelectPadding@; text-align:left;
 SelectDropDown              = background:@WidgetBgColor@; @LeftSeparator@; padding:0 @SelectPadding@;
-SelectBorder                = @TextAreaBorder@ @MediumRoundCorners@
+SelectBorder                = @TextAreaBorder@ @MediumRoundCorners@ background:@white@;
```

Comment:
Though `@appc@` value is white, but I don't feel it should be used. We need `white` color only. `@appc@` may be of different color in some child theme.